src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -532,10 +536,11 @@ public void should_return_a_200_with_search_results_when_role_assignment_has_no_
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.size()", lessThanOrEqualTo(10))
-            .body("tasks.location", everyItem(equalTo("765324")))
-            .body("tasks.jurisdiction", everyItem(is("IA")))
-            .body("total_records", greaterThanOrEqualTo(1));
+            .body("tasks.size()", equalTo(1))
+            .body("tasks[0].id", equalTo(taskId))
```

Comment:
Same as above, are we guaranteed to only return this task always?

Reply:
Yes, always it will return a single record because of the search filter caseId.